src/services/explore.js

```diff
@@ -4,7 +4,7 @@ import request from '../utils/request';
 export async function querySharedSessions(params) {
   const { datastoreConfig } = params;
 
-  const endpoint = `${datastoreConfig.graphql}`;
+  const endpoint = `${datastoreConfig.pbench_server}/graphql`;
```

Comment:
Similar to the above comment. Please mention in the PR if we have to add new fields to our `datastoreConfig`